packages/devtools_app/lib/src/screens/memory/memory_tracker_model.dart

```diff
@@ -141,7 +142,8 @@ class TreeTracker {
     Map<String, ClassRef> classesTracked,
     List<AllocationSamples> allocationSamples,
   ) {
-    final tree1Local = tree1 = TrackerClass('_ROOT_tracking');
+    tree1 ??= TrackerClass('_ROOT_tracking');
+    final tree1Local = tree1!;
```

Comment:
General dart null safety comment - the fact that we have to null assert here is surprising 🤔

Reply:
`tree` isn't a local property so it could, in theory, be set to null again between the allocation and the assignment to `tree1Local` if there was any sort of asynchrony between those lines. There obviously can't be an asynchronous gap here in this case, but I'm guessing the CFE can't detect that for one reason or another.